.travis.yml

```diff
@@ -14,7 +14,7 @@ matrix:
       env:
         - USE_SYSTEM=1
         - USE_PROJECT=1
-        - BLASLAPACK_IMPLEMENTATION=OpenBLAS
+        - BLASLAPACK_IMPLEMENTATION=vecLib
```

Comment:
are you sure? what is the default implem for mac? the test config about uses default on mac. Should I understand that OpenBLAS is the default on mac now?